.gitignore

```diff
@@ -6,6 +6,8 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+./config/keys.js
```

Comment:
why are the keys file in a javascript file and not .env?

Reply:
fixed